<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvantagesAndDisadvantages</class>
 <widget class="QWidget" name="AdvantagesAndDisadvantages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>607</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="Merits" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Merits</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addNewWidgetButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteWidgetButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="Merit1" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="MeritsDetail1">
         <item>
          <layout class="QHBoxLayout" name="MeritsNameDots">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLineEdit" name="meritLine">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>23</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="LoresheetButtonsLayout_1">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <item>
              <widget class="QRadioButton" name="loresheetButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">MeritButtonGroup1</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="loresheetButton_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">MeritButtonGroup1</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="loresheetButton_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">MeritButtonGroup1</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="loresheetButton_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">MeritButtonGroup1</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="loresheetButton_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">MeritButtonGroup1</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="Flaws" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Flaws</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="addNewWidgetButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteWidgetButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="Flaw1" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="FlawsDetail1">
         <item>
          <layout class="QHBoxLayout" name="LoresheetNameDots_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLineEdit" name="flawLine">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>23</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <bold>false</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="FlawsButtonsLayout_2">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <item>
              <widget class="QRadioButton" name="loresheetButton_6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">FlawButtonGroup1</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="loresheetButton_7">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">FlawButtonGroup1</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="loresheetButton_8">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">FlawButtonGroup1</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="loresheetButton_9">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">FlawButtonGroup1</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="loresheetButton_10">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton::indicator{

width: 20px;
height: 20px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">FlawButtonGroup1</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="lockButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/icons/unlock20x20.png</normaloff>:/images/icons/unlock20x20.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>66</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="MeritButtonGroup1">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="FlawButtonGroup1">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
